src/gtc/utils.py

```diff
@@ -38,3 +38,9 @@ def flatten_list_iter(
         else:
             if item is not None or not filter_none:
                 yield item
+
+
+def dimension_flags_to_names(mask: Tuple[bool, bool, bool]) -> str:
+    labels = ["i", "j", "k"]
```

Comment:
I'm not a fan of the `LatLonGrid` class, but for the sake of consistency should the names come from the same places as previously:
```Domain.LatLonGrid().axes_names```

Reply:
Since this PR is about introducing gtc support for lower dims I refuse to increase dependencies on the old toolchain here. Can be done somewhere else if justified.

Reply:
Sure, I have added a comment to remove the dependency on `LatLonGrid` above.